website/docs/r/buildpack.html.markdown

```diff
@@ -30,7 +30,7 @@ resource "cf_buildpack" "tomee" {
 The following arguments are supported:
 
 * `name` - (Required) The name of the Buildpack.
-* `position` - (Optional, Number) Specifies where to place the buildpack in the detection priority list. For more information, see the [Buildpack Detection](https://docs.cloudfoundry.org/buildpacks/detection.html) topic.
+* `position` - (Optional, Number) Specifies where to place the buildpack in the detection priority list. For more information, see the [Buildpack Detection](https://docs.cloudfoundry.org/buildpacks/detection.html) topic. When not provided, value is computed and retreived from Cloud Foundry.
```

Comment:
suggesting: 

When not provided, cloudfoundry assigns a default buildpack position.

(I looked up Cf CLI docs and code and could not find code assigning default position. [CC API](http://apidocs.cloudfoundry.org/1.27.0/buildpacks/creates_an_admin_buildpack.html) does not detail it either default value. Would need to look into cloud controller tests)